.github/workflows/notify-slack.yml

```
@@ -0,0 +1,65 @@
+---
+
+name: 'Notify slack'
+
+on:
+  workflow_call:
+    inputs:
+      application:
+        description: 'The friendly name for the application'
+        type: string
+        required: true
+
+      environment:
+        description: |
+          The environment variables to pass to the render.  Each environment variable should be in format VAR=VAL with one on each line.
+          Example:
+            environment: |
+              FOO=bar
+              BAR=baz
+        type: string
+        required: false
+        default: ''
+
+      template:
+        description: 'The path to the template file to render'
+        type: string
+        required: true
+
+      version:
+        description: 'The version that is being deployed'
+        type: string
+        default: ${{ github.ref }}
+
+      type:
+        description: 'The type of action that is being performed such as build or deployment'
+        type: string
+        required: true
+
+      status:
+        description: 'The status of the deployment.  Options are started, success, and failure'
+        type: string
+        required: true
+
+    secrets:
+      slack-webhook-url:
+        description: 'The slack webhook url to post the message to'
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  notify-slack:
+    name: 'Publish slack notification'
+    uses: ./.github/actions/notify-slack
+    with:
+      application: ${{ inputs.application }}
+      environment: ${{ inputs.environment }}
+      status: ${{ inputs.status }}
+      type: ${{ inputs.type }}
+      template: ${{ inputs.template }}
+      version: ${{ inputs.version }}
+    secrets:
+      slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
```

This should be the secret provided to the workflow_call, not the repo secret.

```suggestion
      slack-webhook-url: ${{ secrets.slack-webhook-url }}
```